docs/project-model/global-json-reference.md

```diff
@@ -0,0 +1,24 @@
+# Global.json reference
+
+* [projects/sources](#projects)
+* [packages](#packages)
+
+<a name="projects"></a>
+## projects
+An array of string that specifies what folders the build system should search for projects before falling back to resolving dependencies via nuget
```

Comment:
I don't think this quite captures what the purpose is here.  For example, if I have this:

```json
   "dependencies":{
      "MyDependencyClassLibrary":""
   }
```
There's nothing for NuGet to resolve.  I'd stick to just stating that this is for the build system to understand where you've placed projects you depend on, should you choose to organize things in a way that differs from the (hopefully eventualy?) convention that doesn't require any configuration.